geek.txt/backend/server.js

```diff
@@ -18,15 +18,15 @@ app.use(cors());
 app.use(express.json());
 
 // Load ATLAS_URI environment variable.
-const uri = process.env.ATLAS_URI
+// const uri = process.env.ATLAS_URI
 
 // Connect to the remote database.
-mongoose.connect(uri, {useNewUrlParser: true, useCreateIndex: true, useUnifiedTopology:true});
+// mongoose.connect(uri, {useNewUrlParser: true, useCreateIndex: true, useUnifiedTopology:true});
 
-const connection = mongoose.connection;
-connection.once('open', () => {
-    console.log("Geek.txt MongoDB is connected");
-});
+// const connection = mongoose.connection;
```

Comment:
why is the DB connection commented out?

Reply:
Everyone doesn't have the URI string or the .env file and this would cause the backend to crash. Until everyone informed about it or has the db string best to leave it commented out. Also, the db is not being used currently.